book/src/graffiti.md

```diff
@@ -0,0 +1,58 @@
+# Validator graffiti
+
+Lighthouse provides 3 options for setting validator graffitis.
+
+### Using the "--graffiti-file" flag on the validator client
+Users can specify a file with the `--graffiti-file` flag. This option is useful for dynamically changing graffitis for various use cases (e.g. drawing on the beaconcha.in graffiti wall). This file is loaded once on startup and reloaded everytime a validator is chosen to propose a block.
+
+Usage:
+`lighthouse vc --graffiti-file graffiti_file.txt`
+
+The file should contain key value pairs corresponding to validator public keys and their associated graffiti. The file should start with a default key for the  default graffiti case.
```

Comment:
```suggestion
The file should contain key value pairs corresponding to validator public keys and their associated graffiti. The file must start with a default key for the default graffiti case.
```

(Assuming we don't allow the default line to appear anywhere in the file).

Reply:
Allowed graffiti to appear anywhere